server/app/uk/gov/ons/addressIndex/server/controllers/EQController.scala

```diff
@@ -36,25 +36,14 @@ class EQController @Inject () (val controllerComponents: ControllerComponents,
               groupfullpostcodes: Option[String] = None
              ): Action[AnyContent] = Action async { implicit req =>
 
-    val groupFullPostcodesVal: Boolean = Try(groupfullpostcodes.getOrElse("false").toBoolean).toOption.getOrElse(false)
+ // groupfullpostocdes = yes, no, combo
```

Comment:
I would have looked at using an enum rather than a string for yes/no/combo. Just my personal choice  :) would  approach the enum using sealed case objects as shown here https://docs.scala-lang.org/overviews/scala-book/enumerations-pizza-class.html

Reply:
I'll have a look at doing an enum